src/FluentMigrator.Extensions.Postgres/Postgres/PostgresExtensions.IndexStorage.cs

```diff
@@ -44,38 +43,5 @@ public static ICreateBTreeIndexOptionsSyntax Fillfactor(this ICreateIndexOptions
             return expression.UsingBTree()
                 .Fillfactor(fillfactor) as ICreateBTreeIndexOptionsSyntax;
         }
-
```

Comment:
@lillo42 Why is this PR about improving the docs but you're deleting code?  Everything else looks good to me.  Just making sure you did this on purpose.

Reply:
@lillo42  Any answer here?  Thanks.

Reply:
I'm curious about this change too. However, I do like that the constants are moved to the `PostgresExtensions` file even though that it'll be a breaking change too (at source level).

Reply:
Sorry for delay, I was moving last week, a lot of thing to do, about the delete code I realize that I have forgot to delete this code after move the storage parameter to the interfaces, maybe it's better rollback it and add Obsolete attribute, what do you think?

Reply:
If there is a replacement for those functions and it didn't exist for too long, then I agree that the breaking change might not be too disruptive for the FM users out there.